security/interceptor/security-interceptor-anonymous/src/main/java/org/codice/ddf/security/interceptor/AnonymousInterceptor.java

```diff
@@ -286,15 +303,10 @@ public void handleMessage(SoapMessage message) throws Fault {
                                     String xpathStrict = "/Layout/Policy/Strict";
                                     String xpathLaxTimestampFirst = "/Layout/Policy/LaxTimestampFirst";
                                     String xpathLaxTimestampLast = "/Layout/Policy/LaxTimestampLast";
-                                    layoutLax = evaluateExpression(xml, xpathLax);
-                                    layoutStrict = evaluateExpression(xml, xpathStrict);
-                                    layoutLaxTimestampFirst = evaluateExpression(xml, xpathLaxTimestampFirst);
-                                    layoutLaxTimestampLast = evaluateExpression(xml, xpathLaxTimestampLast);
 
```

Comment:
This code is half finished.  Eventually we will need these xpath values when we inspect the policy so I left the values for now.  I did remove the xpath evaluation for now since it was slowing down every request and then not being used.